lib/ci/lib/rails_upgrade_analyzer.rb

```diff
@@ -0,0 +1,11 @@
+require "octokit"
+require "travis"
+
+$LOAD_PATH.unshift(File.dirname(__FILE__))
```

Comment:
`lib` should already be in the load path`.  Why are we nesting another `lib` inside it?